cmd/admin-rpc-client.go

```diff
@@ -65,20 +65,20 @@ func (rc remoteAdminClient) Restart() error {
 	return err
 }
 
-type controlPeer struct {
+type adminPeer struct {
```

Comment:
We should write some comments about the actual usage of these peers and the structure used here.